content/ja/docs/contributing/sig-practices.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 承認者およびメンテナーのための SIG のプラクティス
+linkTitle: SIG のプラクティス
+description: 承認者およびメンテナーがどのようにイシューやコントリビューションを管理するかを学びます。
+weight: 999
+default_lang_commit: 722beb5b9ecf8a603b2f5ae6128e3c3509f52fcf
+cSpell:ignore: chalin docsy
+---
+
+このページでは、承認者およびメンテナーが使用するガイドラインと一般的なプラクティスについて説明します。
+
+## オンボーディング {#onboarding}
+
+コントリビュータがドキュメントに対する責任の大きい役割（承認者、メンテナー）を引き受ける場合、既存の承認者およびメンテナーによってオンボーディングが行われます。
+
+- `docs-approvers`（または `docs-maintainers`）グループに追加されます
+- `#otel-comms`、`#otel-maintainers`、およびチーム内のプライベート Slack チャンネルに追加されます
+- [SIG Comm ミーティング](https://groups.google.com/a/opentelemetry.io/g/calendar-comms)と[メンテナーミーティング](https://groups.google.com/a/opentelemetry.io/g/calendar-maintainer-meeting)のカレンダー招待への登録を求められます
+- SIG Comm の現在のミーティング時間が適しているかを確認し、適さない場合は既存の承認者およびメンテナーと調整して、全員に適した時間を決定します
+- コントリビュータ向けのさまざまなリソースを確認します。
```

Comment:
```suggestion
- コントリビュータ向けのさまざまなリソースを確認します
```

Reply:
@ymotongpoo

Thank you for your review.
I fixed it in a below commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6538/commits/a2383370f7db68a4966ba532a9179f0b2a3ae609